hazelcast-sql/src/test/java/com/hazelcast/jet/sql/impl/connector/jdbc/PredicatePushDownJdbcSqlConnectorTest.java

```diff
@@ -54,9 +54,9 @@ public static Object[] parameters() {
         return new Object[]{
                 // Logical operators
                 "SELECT name FROM people WHERE name = 'John Doe'",
-                "SELECT name FROM people WHERE a AND b",
-                "SELECT name FROM people WHERE a OR b",
-                "SELECT name FROM people WHERE NOT c",
```

Comment:
hmm, I think those are valid queries. If it's not valid for MS SQL, we should make an exception for MS SQL only (or perform some kind of conversion)

Reply:
These tests test that each of the operator can be pushed down to the database and executed there.

It was using a BOOLEAN column, because it was the easiest to do, but using any kind of boolean expression as operand is ok as well. This way we keep the same test for all DBs